### DISCUSSION TITLE
Discord Hosting

### DISCUSSION THREAD

**Discussion by stainowy** [community/community#72961] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

EN = Does anyone of you know if I can host small python files on github? These are files for the discord bot

PL = Czy wie ktos z was czy moge zahostowac małe pliki python na github? Są to pliki do bota discord

**Comment by subidit** [comment]
You can try [gist](https://gist.github.com/).

**Comment by DANLOANS** [comment]
If it's about discord hosting, I recommend you to use Heroku, but if you're not satisfied, try looking for something else.
